core/app/models/spree/order_update_attributes.rb

```diff
@@ -0,0 +1,36 @@
+module Spree
+  class OrderUpdateAttributes
```

Comment:
My understanding was that we'll add more stuff here later but we're starting out w/ the minimum that we need to do the payments stuff that triggered all this.